elex/cli/hooks.py

```diff
@@ -28,6 +30,14 @@ def add_election_hook(app):
     if app.pargs.raceids:
         app.election.raceids = [x.strip() for x in app.pargs.raceids.split(',')]
 
+    if app.pargs.officeids:
+        invalid_officeids = [x.strip() for x in app.pargs.officeids.split(',') if x.strip() not in maps.OFFICE_NAMES]
```

Comment:
Why are the values being `strip`ped? It doesn't look like the processing of the office IDs elsewhere in the code is being stripped. But I'm not super familiar with arg parsing, so defer to you if this is necessary.

Reply:
Nope, you're right. It's unnecessary, particularly since adding a bunch of spaces in front of `P,H,G` in `elex races 2016-11-08 --officeids P,H,G` doesn't cause `officeids` to reflect those spaces, and  `elex races 2016-11-08 --officeids P, H, G` errors out and doesn't run.